Mathlib/RingTheory/Valuation/ValuationRing.lean

```diff
@@ -413,6 +413,36 @@ theorem of_integers : ValuationRing 𝒪 := by
     use c; exact Or.inl hc.symm
 #align valuation_ring.of_integers ValuationRing.of_integers
 
+instance instValuationRingInteger : ValuationRing v.integer :=
+  of_integers (v := v) (Valuation.integer.integers v)
+
```

Comment:
Can you add a reference to this result in the preamble of the file?

Reply:
I added "We also show that, given a valuation `v` on a field `K`, the ring of valuation integers is a
valuation ring and `K` is the fraction field of this ring." at the preamble.